doc/classes/Image.xml

```diff
@@ -407,6 +408,7 @@
 			<param index="0" name="buffer" type="PackedByteArray" />
 			<description>
 				Loads an image from the binary contents of a WebP file.
+				[b]Note:[/b] This method is only available in engine builds with the WEBP module enabled. By default, the WEBP module is enabled, but it can be disabled at build-time using the [code]module_webp_enabled=no[/code] SCons option.
```

Comment:
```suggestion
				[b]Note:[/b] This method is only available in engine builds with the WEBP module enabled. By default, the WebP module is enabled, but it can be disabled at build-time using the [code]module_webp_enabled=no[/code] SCons option.
```